components/org.wso2.carbon.identity.application.authenticator.hypr.common/src/main/java/org/wso2/carbon/identity/application/authenticator/hypr/common/model/RequestIDResponse.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2023, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.application.authenticator.hypr.common.model;
+
+/**
+ * Model class for the sub response that contains request ID received upon successfully sending a push notification
+ * from HYPR platform.
+ */
+public class RequestIDResponse {
+
+    private String requestId;
+
+    public String getRequestId() {
+        return this.requestId;
+    }
+
+    public void setRequestId(final String requestId) {
```

Comment:
Add a newline after the method signature.

Reply:
Fixed with [eb39195](https://github.com/wso2-extensions/identity-outbound-auth-hypr/pull/1/commits/eb39195b388a647223846891193117538a081295)